README.md

```diff
@@ -74,6 +74,26 @@ See the [oauth2 docs][] for complete instructions on using that library.
 For API methods that require HTTP Basic Authentication, use the
 [`BasicAuthTransport`](https://godoc.org/github.com/google/go-github/github#BasicAuthTransport).
 
+GitHub Apps authentication can be provided by [ghinstallation](https://github.com/bradleyfalzon/ghinstallation).
+
+```go
+import "github.com/bradleyfalzon/ghinstallation"
+
+func main() {
+  // Shared transport to reuse TCP connections.
+  tr := http.DefaultTransport
```

Comment:
I think you can delete this line and its comment, and pass `http.DefaultTransport` directly to the `ghinstallation.NewKeyFromFile` call below.

It's pretty common knowledge that transports should be reused, unless there are specific reasons not to. I don't think you need to elaborate on that here. This is meant to be a minimal simple example to get started quickly.